Solutions/Corvus.Extensions.Newtonsoft.Json/Microsoft/Extensions/DependencyInjection/JsonSerializerSettingsProviderServiceCollectionExtensions.cs

```diff
@@ -21,27 +23,142 @@ public static class JsonSerializerSettingsProviderServiceCollectionExtensions
         /// </summary>
         /// <param name="services">The target service collection.</param>
         /// <returns>The service collection.</returns>
+        /// <remarks>
+        /// <para>
+        /// Calling this method is now equivalent to calling the following methods:
```

Comment:
The "now" made me think you'd changed the behaviour. (It does this now? What did it use to do before?)

After looking at the history I've discovered that the thing that has changed is that you now have alternative ways to do some or all the things this used to do in one shot. So this thing in effect does what it always did, it's just that that's now probably not what you want.

I'd consider deleting the word "now" from this sentence, and then adding a sentence after that saying "This is now obsolete because `AddJsonNetDateTimeOffsetConverter` adds behaviour designed to work around a limitation that CosmosDB once had, and which can cause problems in general use. Applications should now..." (and then the text that would follow depends on how you respond to my suggestion that we might actually want to add a new method that registers everything except the `DateTimeOffset` converter).

Reply:
I've updated the comment to more accurately reflect what this does and when you'd want to use it. It's still relevant under certain circumstances, as now described in the remarks.